ceph-maintainer-tool/build/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# the following two methods exist in scripts/build_utils.sh
+pkgs=( "python-ldap" )
+TEMPVENV=$(create_venv_dir)
+VENV=${TEMPVENV}/bin
+
+source $VENV/activate
```

Comment:
don't think we need to source the venv, and when I tried to execute this on the jenkins, it failed

Reply:
Removed source venv.